templates/named.zone.erb

```diff
@@ -30,3 +30,4 @@ zone "<%= @zone %>" {
     notify <%= @_dns_notify %>;
 <% end -%>
 };
+
```

Comment:
Probably unintentional, but I will check tomorrow how the config looks like without the newline, and remove it then.